tools/harness-simulation/posix/launch_testbed.py

```diff
@@ -131,53 +128,41 @@ def main():
 
     # Parse arguments
     parser = argparse.ArgumentParser()
-
-    # Determine the interface
-    parser.add_argument('-i',
-                        '--interface',
-                        dest='ifname',
+    parser.add_argument('-c',
+                        '--config',
+                        dest='config',
                         type=str,
                         required=True,
-                        help='the interface used for discovery')
-
-    # Determine the number of OpenThread FTD simulations to be "detected" and then started
-    parser.add_argument('--ot',
-                        dest='ot_num',
-                        type=int,
-                        required=False,
-                        default=0,
-                        help=f'the number of OpenThread FTD simulations')
-
-    # Determine the number of OpenThread BR simulations to be initiated and then detected
-    parser.add_argument('--otbr',
-                        dest='otbr_num',
-                        type=int,
-                        required=False,
-                        default=0,
-                        help=f'the number of OpenThread BR simulations')
-
-    # Determine the number of sniffer simulations to be started and then detected
-    parser.add_argument('--sniffer',
-                        dest='sniffer_num',
-                        type=int,
-                        required=False,
-                        default=0,
-                        help=f'the number of sniffer simulations, no more than {MAX_SNIFFER_NUM}')
-
+                        help='the path of the configuration JSON file')
     args = parser.parse_args()
+    with open(args.config, 'r') as f:
+        config = json.load(f)
+
+    ot_path = config['ot_path']
+    ot_build = config['ot_build']
+    max_nodes_num = ot_build['max_number']
+    # No test case requires more than 2 sniffers
+    max_sniffer_num = 2
```

Comment:
```suggestion
    MAX_SNIFFER_NUM = 2
```
it looks like a constant

Reply:
I use lowercase to keep consistent with other variables like `max_nodes_num`. But `MAX_SNIFFER_NUM` is assigned directly with a constant instead of a value retrieved from the configuration file. Perhaps uppercase should be a better choice for `MAX_SNIFFER_NUM`.